index/borrow.php

```diff
@@ -7,7 +7,8 @@
 <script src="/assets/js/bootstrap.min.js"></script>
 <script src="/assets/js/self-borrow.js"></script>
 <?php
-	require '../assets/api/checkuser.php';
+	require_once('../assets/api/dbconnector.php');
+	require_once '../assets/api/checkuser.php';
```

Comment:
According to "Pear Coding Standards", we don't need to add parenthesis after `require_once`. I think it's a good idea to keep the same style.
[1] http://pear.php.net/manual/en/standards.including.php